docs/installation/installing-with-npm.md

```diff
@@ -208,15 +200,19 @@ a code sample you could add to your configuration:
 ```JS
 app.use('/assets', express.static(path.join(__dirname, '/node_modules/govuk-frontend/assets')))
 ```
+#### Alternative solution
+
+Manually copy the images and fonts from `/node_modules/govuk-frontend/assets` into a public facing directory in your project. Ideally copying the files to your project should be an automated task or part of your build pipeline to ensure that the GOV.UK Frontend assets stay up-to-date.
+
+The default paths used for assets are `assets/images` and `assets/fonts`. **If your asset folders follow this structure, you will not need to complete the following steps.**
+
+To use different asset paths, also complete the below step(s).
 
-#### Alternative solution:
+1. Set `$govuk-assets-path`, `$govuk-images-path` and `$govuk-fonts-path` in your project Sass file to point to the relevant directories in your project (this will override the defaults set in `/node_modules/govuk-frontend/settings/_assets.scss`). Make sure you do this in Sass before importing `govuk-frontend` into your project - see [Importing styles](#importing-styles).
```

Comment:
I don't know if it's clear from this that you either need to set `$govuk-assets-path` OR (`$govuk-images-path` and `$govuk-fonts-path`) – as long as both the images and fonts are in a folder within a common assets folder the user should only need to change the first one.

Reply:
Now with added examples (thanks @36degrees)